pkg/cmd/cli/uninstall/uninstall.go

```diff
@@ -19,96 +19,157 @@ package uninstall
 import (
 	"context"
 	"fmt"
-	"strings"
 	"time"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	kubeerrs "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
 
 	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 
 	"github.com/vmware-tanzu/velero/pkg/client"
 	"github.com/vmware-tanzu/velero/pkg/cmd"
-	"github.com/vmware-tanzu/velero/pkg/cmd/util/output"
+	"github.com/vmware-tanzu/velero/pkg/cmd/cli"
 	"github.com/vmware-tanzu/velero/pkg/install"
 	"github.com/vmware-tanzu/velero/pkg/util/kube"
 )
 
-// Uninstall uninstalls all components deployed using velero install command
-func Uninstall(ctx context.Context, client *kubernetes.Clientset, extensionsClient *apiextensionsclientset.Clientset, veleroNamespace string) error {
-	if veleroNamespace == "" {
-		veleroNamespace = "velero"
-	}
-	err := DeleteNamespace(ctx, client, veleroNamespace)
-	if err != nil {
-		return errors.WithMessagef(err, "Uninstall failed removing Velero namespace %s", veleroNamespace)
+// uninstallOptions collects all the options for uninstalling Velero from a Kubernetes cluster.
+type uninstallOptions struct {
+	wait, force bool
+}
+
+// BindFlags adds command line values to the options struct.
+func (o *uninstallOptions) BindFlags(flags *pflag.FlagSet) {
+	flags.BoolVar(&o.wait, "wait", o.wait, "Wait for Velero deployment to be ready. Optional.")
```

Comment:
Let's change this to "Wait for Velero uninstall to complete. Optional." or something similar.